src/feed/Feed.js

```diff
@@ -177,6 +176,9 @@ class Feed {
         callback({ quotes });
 
         this._emitDataUpdate(quotes, comparisonChartSymbol);
+        if (this.endEpoch) {
+            this._stx.home();
```

Comment:
to make sure, on `historical data` graph always scroll to right side on each time select